api/src/main/resources/client_api.yaml

```diff
@@ -190,6 +184,37 @@ definitions:
         items:
           type: string
 
+  ResultFilters:
```

Comment:
opt: perhaps this should just be called `Predicate` or similar at this point as it now support arbitrary boolean expressions

Reply:
While I agree "predicate" sorta makes sense, it feels more geeky / less researcher-friendly to me than "filter". 

I also wanted some prefix to make it clear this applies to the results that come back and to distinguish it from other places in our API that might use predicates / filters.

(I'm not strongly attached to the name, just explaining how I landed on this.)